.codex/skills/test-vela-extension/scripts/trigger_scan.mjs

```
@@ -0,0 +1,98 @@
+const devtoolsUrl = process.env.CHROME_DEVTOOLS_URL ?? 'http://127.0.0.1:9222';
```

ESLint

[error] 'process' is not defined.

(no-undef)